### PR TITLE
addicotion sfixes

### PR DESCRIPTION
**Pull request by tralezab**
## Why It's Good For The Game

I still don't get how you keep losing so many accumulators. I build 'em tough, and they're supposed to last. I ran the numbers on the comm speeds, and I'm pretty sure you sent for this order less than two days after you should've got the last one. 

If this keeps up, you're gonna punch a hole in something where it doesn't belong. You've got a real good sponsor, Jim, but this stuff ain't cheap to ship. And it sure as heck ain't fast. So, I'm sending you a custom part. One that's as tough as I can make, and with a safety to keep you from overcharging it - that I'm sure you'll bypass anyway. 

At least try not to blow this one to bits for the first week, alright? Make our hometown look good. You're not gonna show up those fancy Europeans with a smoking scrap heap.

## Changelog
:cl:
fix: Addiction bugs put to rest, finally.
/:cl:

### CODE REVIEW

File: code/__DEFINES/reagents.dm [hunk 1]
```diff
@@ -107,15 +107,17 @@
 ///Minimum requirement for addiction buzz to be met
 #define MIN_ADDICTION_REAGENT_AMOUNT 2
 ///Nicotine requires much less in your system to be happy
-#define MIN_NICOTINE_ADDICTION_REAGENT_AMOUNT 0.1
+#define MIN_NICOTINE_ADDICTION_REAGENT_AMOUNT 0.09
```
Comment (tralezab) [inline code/__DEFINES/reagents.dm:110]: When I checked nicotine, 0.1 was still too high.

**Comment by TheVekter** [comment]
addicotion sfixes

**Comment by Qustinnus** [comment]
I explained to you that this is a shit solution, and yet you add the same changes I removed because they broke more. Why do you think they will now suddenly become better?

**Comment by Mothblocks** [comment]
@Qustinnus can you please say for posterity what you think the correct solution here is, because at this point everyone just wants smoker working already

**Comment by Time-Green** [comment]
Could you make junkie bearable aswell? 5u of drugs every 2 minutes is not remotely fun

**Comment by tralezab** [comment]
honestly i'm busy with other stuff, qust can fix addictions how he wants to

**Comment by Mothblocks** [comment]
i dont think hes going to

**Comment by Time-Green** [comment]
@tralezab would you mind if I just took your code and made a new PR? I really need this to just not be broken, and floyd can always make a better fix later if he actually does

**Comment by Mothblocks** [comment]
the license cant stop him

**Comment by Qustinnus** [comment]
just make a good fix stop being so lazy

**Comment by Time-Green** [comment]
@Qustinnus just make a good fix stop being so lazy

**Pull request closed without merging**